bedrock/firefox/templates/firefox/enterprise/signup-thanks.html

```diff
@@ -16,7 +16,7 @@
 {% endblock %}
 
 {% block body_id %}firefox{% endblock %}
-{% block body_class %}{% endblock %}
+{% block body_class %}mzp-t-firefox{% endblock %}
 
 {% block site_header %}
 <header class="enterprise-head">
```

Comment:
The header here isn't alsigned with the page margins, and the logo could use some breathing space:

![image](https://user-images.githubusercontent.com/400117/60731111-964e1880-9f3e-11e9-906d-0f905ee7d49e.png)